gis/qgis3/Portfile

```diff
@@ -10,7 +10,7 @@ PortGroup           compiler_blacklist_versions 1.0
 
 # For stability reasons and to keep in sync with livecheck, update to latest
 # long-term-release (LTR).
```

Comment:
This PR isn't considering this comment (which admittedly I added myself, see discussion in https://github.com/macports/macports-ports/pull/18253). If really important to have the latest version, however potentially risky in production settings, I'd suggest to create a new port `qgis3-devel`.

Reply:
> This PR isn't considering this comment (which admittedly I added myself, see discussion in #18253). If really important to have the latest version, however potentially risky in production settings, I'd suggest to create a new port `qgis3-devel`.

Alternatively, we can create a (sub-)port for the long term release: `qgis-ltr`, as e.g. [FreeBSD](https://www.freshports.org/graphics/qgis-ltr) have.

Reply:
> Alternatively, we can create a (sub-)port for the long term release: `qgis-ltr`, as e.g. [FreeBSD](https://www.freshports.org/graphics/qgis-ltr) have.

Just added a change to this effect with #19410.